app/forms/idv/send_phone_confirmation_otp_form.rb

```diff
@@ -0,0 +1,106 @@
+module Idv
+  # :reek:InstanceVariableAssumption
+  class SendPhoneConfirmationOtpForm
```

Comment:
Since this is not a user-facing form in the traditional sense, what do you think about making this a service object and dropping the `Form` from the name?

Reply:
Yeah, agree. This started it's life as something that made more sense as a form (it took the delivery method as a param like the form on the send_otp endpoint does) but is pretty far removed from that now.